### DISCUSSION TITLE
Form Elemen: Select do not fill Model value

### DISCUSSION THREAD

**Discussion by ebyon** [orchidsoftware/platform#2595] [General]
Hello,

First of all I would like to thank you to the Orchid community. I love to work with this UI framework.

I have an edit Screen with Select method, but never fills it with the Model value.

If I select and option and click save, the model and database are updated with the right value, but after reload page, select has no selected value.

![image](https://user-images.githubusercontent.com/79396176/234805077-2635aeef-ed84-4aa4-b728-efb8974cdee6.png)

![image](https://user-images.githubusercontent.com/79396176/234804948-6004ce1d-81ed-4207-b0bb-5cc61b6e1d21.png)

Thanks for helping me with that.

**Comment by tabuna** [comment]
The comparison is strict `===`, make sure that the values of the types also match

**Comment by ebyon** [comment]
Thank you!

I generate an associative array with key value pair, and key was integer
even i put "{key}" instead only the numeric value.

I change this to use "strings" as a key and works.

Thank you!

Missatge de Alexandr Chernyaev ***@***.***> del dia dj., 27
d’abr. 2023 a les 23:06:
